system/extensions/host/run-image-boot-sh.toit

```diff
@@ -92,8 +92,15 @@ for (( ; ; )); do
   # Run the image.
   export TOIT_FLASH_UUID_FILE=\$PREFIX/\$current/uuid
   export TOIT_FLASH_REGISTRY_FILE=\$PREFIX/flash-registry
-  (cd \$PREFIX/\$current; ./run-image \$PREFIX/\$current \$PREFIX/scratch)
+  pushd \$PREFIX/\$current
+  ./run-image \$PREFIX/\$current \$PREFIX/scratch &
+  popd
+  RUN_IMAGE_PID=\$!
```

Comment:
I think this should be `run_image_pid` to fit with `exit_code`. Maybe even rename `exit_code` to `run_image_exit_code`?